readwise_vector_db/db/database.py

```
@@ -1,67 +1,141 @@
+
+        AsyncSessionLocal = sessionmaker(
+            engine, 
+            class_=AsyncSession, 
+            expire_on_commit=False
+        )
 
 
-async def get_session() -> AsyncSession:
+async def get_session() -> AsyncGenerator[AsyncSession, None]:
+    """Get an async database session.
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)